addons/medical/functions/fnc_actionCheckResponse.sqf

```diff
@@ -7,22 +7,20 @@
  * 1: The patient <OBJECT>
  *
  * Return Value:
- * NONE
+ * None
  *
  * Public: No
  */
 
 #include "script_component.hpp"
 
-private ["_caller","_target", "_output"];
-_caller = _this select 0;
-_target = _this select 1;
+private ["_output"];
+params ["_caller", "_target"];
 
-_output = "";
-if ([_target] call EFUNC(common,isAwake)) then {
-    _output = LSTRING(Check_Response_Responsive);
+_output = if ([_target] call EFUNC(common,isAwake)) then {
```

Comment:
This might be better:
```sqf
_output = [LSTRING(Check_Response_Unresponsive), LSTRING(Check_Response_Responsive)] select ([_target] call EFUNC(common,isAwake));
```